.golangci.yml

```diff
@@ -106,7 +106,8 @@ linters:
     - asciicheck
     - bodyclose
     - cyclop
-    - deadcode
+    # deprecated
+    # - deadcode
```

Comment:
Do you want to just delete these? I don't think there's a reason to leave them in given we don't intend to reenable.

Reply:
ah yes, fixed.